src/pages/gounSeo/Main/Main.js

```diff
@@ -1,9 +1,167 @@
 import React from 'react';
 import './Main.scss';
-
+import Comment from './Comment/Comment';
 class MainGoUn extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      reply: '',
+      commentArr: [
+        { id: 'canon_mj', text: '베이킹 클래스 즐거웠어~' },
+        { id: 'cocoadidi', text: '다음에 또 같이 하자' },
+      ],
+    };
+  }
+
+  handlereplyInput = event => {
+    this.setState({ reply: event.target.value });
+  };
+
+  clickButton = () => {
+    this.setState({
+      commentArr: this.state.commentArr.concat({
+        id: 'anna',
+        text: this.state.reply,
+      }),
+    });
+    this.resetEnter();
```

Comment:
함수를 따로 선언하고 호출하는 것이 아니라, 이 함수 안에서 다음처럼 한번에 작성할 수 있겠네요.
```suggestion
    this.setState({
      commentArr: this.state.commentArr.concat({
        id: 'anna',
        text: this.state.reply,
      }),
      reply: ''
    });
```

Reply:
그리고 clickButton이라는 함수명은 함수의 역할을 나타내는 것이 아니라, 함수를 호출하기 위한 이벤트를 나타내는 듯한 명칭이네요.

함수의 명확한 행동을 나타내는 명칭으로 지어주시는 것이 좋습니다.

ex) addComment